website/functions.php

```diff
@@ -680,6 +680,21 @@ function wbName($id,$isSuper=false) {
     return $HERO[$id]["name"];
 }
 
+function wbHitsRequired($isSuper=false) {
+    global $sql;
+	$res = $sql->query("SELECT bid, COUNT(DISTINCT uid) AS p FROM WBD GROUP BY bid ORDER BY bid DESC LIMIT 20");
+	$avg = 0;
+	while ($row=$res->fetch_assoc()) {
+		$avg += $row["p"];
+	}
+	$avg = ceil($avg/20);
+	return $avg * ($isSuper ? 2 : 3);
+}
+
+function wbRewardModifier() {
+	return wbHitsRequired() * 3 / 1600;
```

Comment:
The *3 is already included in wbHitsRequired()
```suggestion
	return wbHitsRequired()  / 1600;
```